<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationSettingsDialog</class>
 <widget class="QDialog" name="ConfigurationSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="webDashboardCombo">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>191</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>No</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Yes</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="ImportConfigLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>217</y>
     <width>105</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Import config:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="configParamsGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>25</y>
     <width>361</width>
     <height>138</height>
    </rect>
   </property>
   <property name="title">
    <string>Config Parameters</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="4" column="0">
     <widget class="QLabel" name="flushDelayLabel">
      <property name="text">
       <string>Flush stats after:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="logLocationLabel">
      <property name="text">
       <string>Log file path</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="thresholdLimitLabel">
      <property name="text">
       <string>Threshold:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="refreshDelayLabel">
      <property name="text">
       <string>Refresh Log after:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="loglocationCombo">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="cursor">
       <cursorShape>IBeamCursor</cursorShape>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="browseLogFileButton">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QSpinBox" name="thresholdLimitSpinner">
      <property name="suffix">
       <string/>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>1000000</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QSpinBox" name="refreshDelaySpinner">
      <property name="suffix">
       <string>s</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10000000</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QSpinBox" name="flushDelaySpinner">
      <property name="suffix">
       <string>s</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10000000</number>
      </property>
      <property name="value">
       <number>120</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="webDashboardLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate Web Dshboard:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="settingWarningsLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="browseConfigPathButton">
   <property name="geometry">
    <rect>
     <x>314</x>
     <y>220</y>
     <width>61</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QComboBox" name="configPathCombo">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="applicationConfigGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>361</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Application config</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_5"/>
  </widget>
  <zorder>applicationConfigGroupBox</zorder>
  <zorder>webDashboardCombo</zorder>
  <zorder>ImportConfigLabel</zorder>
  <zorder>configParamsGroupBox</zorder>
  <zorder>webDashboardLabel</zorder>
  <zorder>saveButton</zorder>
  <zorder>cancelButton</zorder>
  <zorder>settingWarningsLabel</zorder>
  <zorder>browseConfigPathButton</zorder>
  <zorder>configPathCombo</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
